nixos/modules/services/web-servers/gunicorn/default.nix

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
I really think this should be it's own PR... It sucks a bit to split it when one PR depends on the other, meaning one kinda have to wait for the other. But I think it would be good to have some more eyes on gunicorn in particular, because it will be used by other modules around in nixpkgs.

In the meanwhile, maybe it would be possible to have uwsgi host otterwiki instead if you don't want to wait? There's already a reasonably fleshed out module for uwsgi, and I think it has everything you need.